book/src/state_specific_systems/how_to_define_state_dispatcher_internal.md

```diff
@@ -88,9 +88,15 @@ This guide explains how to define a state-specific dispatcher whose systems are
             data.data.update(&data.world);
             self.dispatcher.as_mut().unwrap().dispatch(&data.world.res);
 
-            Trans::Pop
+            Trans::None
         }
     }
     ```
 
     Now, any systems in the state-specific dispatcher will only run in this state.
+
+    Depending on the trait implemented by `CustomDispatcherState` it is recommended to omit the execution of the application dispatcher during `update(..)`:
```

Comment:
Instead of this, I think it would be easier if you assumed SimpleState in this article as well. Not-simple states aren't much different, SimpleState is only here to not have to explain dispatcher updates to beginners.

Reply:
Ahh, haven't reworked the documentation for internal dispatchers yet. Ignore that bit, it's old!